web/src/components/layout/scaffold/Scaffold.vue

```diff
@@ -42,6 +46,8 @@ const props = withDefaults(defineProps<Props>(), {
   // eslint-disable-next-line vue/no-boolean-default
   enableTabs: false,
   activeTab: '',
+  // eslint-disable-next-line vue/no-boolean-default
+  noFluidWrapper: false,
```

Comment:
```suggestion
  fluid: true,
```

I would just call it `fluid`.

Anyways if you have a boolean prop it will automatically be set to true.

Reply:
How about `fluidContent`?